apps/client/src/app/dashboard/page.tsx

```diff
@@ -1,19 +1,11 @@
-import { currentUser } from "@clerk/nextjs/server";
 import { Separator } from "@radix-ui/react-select";
```

Comment:
### Critical Points:
❌ The code now lacks user authentication. The `currentUser` function and the `Notauthorized` component have been removed, meaning anyone can access the dashboard page.

### Good Points:
👍 The code is more concise and streamlined by removing unnecessary code related to user authentication.